app/package.json

```diff
@@ -84,6 +84,7 @@
     "babel-eslint": "^8.2.2",
     "babel-jest": "^23.6.0",
     "babel-loader": "^7.1.2",
+    "babel-plugin-dynamic-import-node": "^2.2.0",
```

Comment:
IIUC this wouldn't be required if we let webpack handle imports rather than babel, right?

Reply:
I'm not sure about that. All I know is that `node` does not recognize dynamic `import` syntax (`npm test`):

<img width="647" alt="screenshot 2018-12-03 12 04 19" src="https://user-images.githubusercontent.com/320066/49370174-bb05b780-f6f3-11e8-96ad-cae6457f910f.png">

…and adding `dynamic-import-node` fixes it.